### ISSUE TITLE
evaluate_ruby_response causes duplicate key warnings

### ISSUE THREAD

**Issue by jcoyne** [rsolr/rsolr#102]
With ruby 2.2
```
(byebug) n
(eval):1: warning: duplicated key at line 1 ignored: "active_fedora_model_ssi"
(eval):1: warning: duplicated key at line 1 ignored: "location_label_sim"
(eval):1: warning: duplicated key at line 1 ignored: "creator_label_sim"
(eval):1: warning: duplicated key at line 1 ignored: "lc_subject_label_sim"
(eval):1: warning: duplicated key at line 1 ignored: "publisher_sim"
(eval):1: warning: duplicated key at line 1 ignored: "year_iim"
(eval):1: warning: duplicated key at line 1 ignored: "form_of_work_label_sim"
(eval):1: warning: duplicated key at line 1 ignored: "collection_label_ssim"
```

```
"{'responseHeader'=>{'status'=>0,'QTime'=>3,'params'=>{'sort'=>'score desc, date_si desc, creator_label_si asc','qf'=>'title_tesim lc_subject_label_tesim','facet.field'=>['active_fedora_model_ssi','location_label_sim','creator_label_sim','lc_subject_label_sim','publisher_sim','year_iim','form_of_work_label_sim','collection_label_ssim'],'qt'=>'search','wt'=>'ruby','fq'=>'has_model_ssim:(\"Image\")','rows'=>'10'}},'response'=>{'numFound'=>1,'start'=>0,'maxScore'=>1.0,'docs'=>[{'system_create_dtsi'=>'2015-03-04T19:18:30Z','system_modified_dtsi'=>'2015-03-04T19:18:30Z','active_fedora_model_ssi'=>'Image','has_model_ssim'=>['Image'],'id'=>'47/3b/6e/c5/473b6ec5-0e40-48a8-96d1-f683c998f3be','object_profile_ssm'=>['{\"id\":\"47/3b/6e/c5/473b6ec5-0e40-48a8-96d1-f683c998f3be\",\"identifier\":[],\"accession_number\":[],\"title\":null,\"alternative\":[],\"creator\":[],\"collector\":[],\"contributor\":[],\"description\":[],\"latitude\":[],\"language\":[],\"longitude\":[],\"location\":[],\"lc_subject\":[],\"publisher\":[],\"work_type\":[],\"series_name\":[],\"issued\":[],\"issued_start\":[],\"issued_end\":[],\"created_start\":[],\"created_end\":[],\"date_other\":[],\"form_of_work\":[],\"citation\":[],\"digital_origin\":[],\"note\":[],\"description_standard\":[],\"extent\":[],\"sub_location\":[],\"record_origin\":[],\"use_restrictions\":[]}'],'_version_'=>1494741640676376576,'timestamp'=>'2015-03-04T19:18:32.376Z','score'=>1.0}]},'facet_counts'=>{'facet_queries'=>{},'facet_fields'=>{'active_fedora_model_ssi'=>['Image',1],'location_label_sim'=>[],'creator_label_sim'=>[],'lc_subject_label_sim'=>[],'publisher_sim'=>[],'year_iim'=>[],'form_of_work_label_sim'=>[],'collection_label_ssim'=>[],'active_fedora_model_ssi'=>['Image',1],'location_label_sim'=>[],'creator_label_sim'=>[],'lc_subject_label_sim'=>[],'publisher_sim'=>[],'year_iim'=>[],'form_of_work_label_sim'=>[],'collection_label_ssim'=>[]},'facet_dates'=>{},'facet_ranges'=>{},'facet_intervals'=>{}}}\n"
```

**Comment by jcoyne** [comment]
Looks like this could be a Solr bug:
```
"{'responseHeader'=>{'status'=>0,'QTime'=>3,'params'=>{'sort'=>'score desc, date_si desc, creator_label_si asc','qf'=>'title_tesim lc_subject_label_tesim','facet.field'=>['active_fedora_model_ssi','location_label_sim','creator_label_sim','lc_subject_label_sim','publisher_sim','year_iim','form_of_work_label_sim','collection_label_ssim'],'qt'=>'search','wt'=>'ruby','fq'=>'has_model_ssim:(\"Image\")','rows'=>'10'}},'response'=>{'numFound'=>1,'start'=>0,'maxScore'=>1.0,'docs'=>[{'system_create_dtsi'=>'2015-03-04T19:18:30Z','system_modified_dtsi'=>'2015-03-04T19:18:30Z','active_fedora_model_ssi'=>'Image','has_model_ssim'=>['Image'],'id'=>'47/3b/6e/c5/473b6ec5-0e40-48a8-96d1-f683c998f3be','object_profile_ssm'=>['{\"id\":\"47/3b/6e/c5/473b6ec5-0e40-48a8-96d1-f683c998f3be\",\"identifier\":[],\"accession_number\":[],\"title\":null,\"alternative\":[],\"creator\":[],\"collector\":[],\"contributor\":[],\"description\":[],\"latitude\":[],\"language\":[],\"longitude\":[],\"location\":[],\"lc_subject\":[],\"publisher\":[],\"work_type\":[],\"series_name\":[],\"issued\":[],\"issued_start\":[],\"issued_end\":[],\"created_start\":[],\"created_end\":[],\"date_other\":[],\"form_of_work\":[],\"citation\":[],\"digital_origin\":[],\"note\":[],\"description_standard\":[],\"extent\":[],\"sub_location\":[],\"record_origin\":[],\"use_restrictions\":[]}'],'_version_'=>1494741640676376576,'timestamp'=>'2015-03-04T19:18:32.376Z','score'=>1.0}]},
'facet_counts'=>{
  'facet_queries'=>{},
  'facet_fields'=>{
    'active_fedora_model_ssi'=>['Image',1],
    'location_label_sim'=>[],
    'creator_label_sim'=>[],
    'lc_subject_label_sim'=>[],
    'publisher_sim'=>[],'year_iim'=>[],
    'form_of_work_label_sim'=>[],
    'collection_label_ssim'=>[],
    'active_fedora_model_ssi'=>['Image',1],
    'location_label_sim'=>[],
    'creator_label_sim'=>[],
    'lc_subject_label_sim'=>[],
    'publisher_sim'=>[],
    'year_iim'=>[],
    'form_of_work_label_sim'=>[],
    'collection_label_ssim'=>[]
  },
  'facet_dates'=>{},'facet_ranges'=>{},'facet_intervals'=>{}}
}\n"
```

**Comment by jcoyne** [comment]
@cbeer says this indeed is a bug in solr which ought to be fixed in 4.10.4

**Comment by cbeer** [comment]
https://issues.apache.org/jira/browse/SOLR-6780
